VPCBuilder/README.md

```diff
@@ -0,0 +1,131 @@
+It will support the removal of Subnets, RouteTables, NATGateways and NetworkACLs.
+
+
+```yaml
+AWSTemplateFormatVersion: 2010-09-09
+Description: Private VPC Template
+Parameters:
+  VGW: {Description: VPC Gateway, Type: String, Default: vgw-a0b1c2d3}
+Mappings: {}
+Resources:
+
+    KABLAMOBUILDVPC:
+        Type: Kablamo::Network::VPC
+        Properties:
+            CIDR: 172.16.0.0/20
+            Details: {VPCName: PRIVATEEGRESSVPC, VPCDesc: Private Egress VPC, Region: ap-southeast-2, IPv6: True}
+            Tags: {Name: PRIVATE-EGRESS-VPC, Template: VPC for private endpoints egress only}            
+            DHCP: {Name: DhcpOptions, DNSServers: 172.16.0.2, NTPServers: 169.254.169.123, NTBType: 2}
+            Subnets:
+                ReservedMgmt1: {CIDR: 172.16.0.0/26, AZ: 0, NetACL: InternalSubnetAcl, RouteTable: InternalRT1 }
+                ReservedMgmt2: {CIDR: 172.16.1.0/26, AZ: 1, NetACL: InternalSubnetAcl, RouteTable: InternalRT2 }
+                ReservedMgmt3: {CIDR: 172.16.2.0/26, AZ: 2, NetACL: InternalSubnetAcl, RouteTable: InternalRT3 }
+                ReservedNet1: {CIDR: 172.16.0.192/26, AZ: 0, NetACL: RestrictedSubnetAcl, RouteTable: PublicRT }
+                ReservedNet2: {CIDR: 172.16.1.192/26, AZ: 1, NetACL: RestrictedSubnetAcl, RouteTable: PublicRT }
+                ReservedNet3: {CIDR: 172.16.2.192/26, AZ: 2, NetACL: RestrictedSubnetAcl, RouteTable: PublicRT }
+                Internal1: {CIDR: 172.16.3.0/24, AZ: 0, NetACL: InternalSubnetAcl, RouteTable: InternalRT1 }
+                Internal2: {CIDR: 172.16.4.0/24, AZ: 1, NetACL: InternalSubnetAcl, RouteTable: InternalRT2 }
+                Internal3: {CIDR: 172.16.5.0/24, AZ: 2, NetACL: InternalSubnetAcl, RouteTable: InternalRT3 }
+                PerimeterInternal1: {CIDR: 172.16.6.0/24, AZ: 0, NetACL: InternalSubnetAcl, RouteTable: InternalRT1 }
+                PerimeterInternal2: {CIDR: 172.16.7.0/24, AZ: 1, NetACL: InternalSubnetAcl, RouteTable: InternalRT2 }
+                PerimeterInternal3: {CIDR: 172.16.8.0/24, AZ: 2, NetACL: InternalSubnetAcl, RouteTable: InternalRT3 }
+            RouteTables:
+                PublicRT:
+                  - RouteName: PublicRoute
+                    RouteCIDR: 0.0.0.0/0
+                    RouteGW: InternetGateway
+                  - RouteName: PublicRouteIPv6
+                    RouteCIDR: ::/0
+                    RouteGW: InternetGateway
+                InternalRT1:
+                InternalRT2:
+                InternalRT3:
+            NATGateways:
+                NATGW1:
+                    {Subnet: ReservedNet1, Routetable: InternalRT1}
+                NATGW2:
+                    {Subnet: ReservedNet2, Routetable: InternalRT2}
+                NATGW3:
+                    {Subnet: ReservedNet3, Routetable: InternalRT3}
+            NetworkACLs:
+                RestrictedSubnetAcl: 
+                    RestrictedSubnetAclEntryInTCPUnReserved: "90,6,allow,false,0.0.0.0/0,1024,65535"
```

Comment:
Can we add some info to the readme, about what these positional args are?

Reply:
Added into the VPC Builder README.md

Reply:
👍